orchestrator/orchestrator/terminal.py

```diff
@@ -84,15 +85,33 @@ def _start_attach(self, command: Optional[List[str]]) -> None:
             self.client, self.site_id, self.site_data
         )
 
+        # Allows for the use of a C program to perform keepalive instead of
+        # using `sh`, better performance
+        if settings.TERMINAL_KEEPALIVE_PROGRAM_PATH is not None:
+            keepalive_command = [
+                "/terminal-keepalive",
+                str(settings.SITE_TERMINAL_KEEPALIVE_TIMEOUT),
+            ]
+            run_params.get("mounts").append(
```

Comment:
```suggestion
            run_params["mounts"].append(
```

`run_params["mounts"]` will *always* be set.